linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml

```diff
@@ -4,25 +4,31 @@
 # complexity = low
 # disruption = low
 
-{{% if 'sle' in product %}}
-- name: Verify that Interactive Boot is Disabled in /usr/bin/grub2-editenv
+- name: Verify GRUB_DISABLE_RECOVERY=true
+  lineinfile:
+    path: /etc/default/grub
+    regexp: '^GRUB_DISABLE_RECOVERY=.*'
+    line: 'GRUB_DISABLE_RECOVERY=true'
+    state: present
+
+- name: Verify that Interactive Boot is Disabled in /etc/default/grub
   replace:
     dest: /etc/default/grub
-    regexp: systemd.confirm_spawn=(1|yes|true|on) 
+    regexp: systemd.confirm_spawn=(1|yes|true|on)
     replace: systemd.confirm_spawn=no
 
+{{% if 'sle' in product %}}
 - name: Verify that Interactive Boot is Disabled (runtime)
   command: /usr/bin/grub2-editenv - unset systemd.confirm_spawn
 
-{{% else %}}
+- name: Verify that Recovery Boot is Disabled (runtime)
+  command: grub2-mkconfig -o /boot/grub2/grub.cfg
 
-- name: Verify that Interactive Boot is Disabled in /etc/default/grub
-  replace:
-    dest: /etc/default/grub
-    regexp: systemd.confirm_spawn=(1|yes|true|on)
-    replace: systemd.confirm_spawn=no
+{{% else %}}
 
 - name: Verify that Interactive Boot is Disabled (runtime)
   command: /sbin/grubby --update-kernel=ALL --remove-args="systemd.confirm_spawn"
 
+- name: Verify that Recovery Boot is Disabled (runtime)
+  command: sudo grubby --update-kernel=ALL --args="GRUB_DISABLE_RECOVERY=true"
```

Comment:
This can't be used with the Grubby. Please use 
grub2-mkconfig -o {{{ grub2_boot_path }}}/grub2.cfg
instead. This holds also for the Bash remediation.

Reply:
Thank, I have updated code to address.